internal/resolver/ooniresolver/ooniresolver_test.go

```diff
@@ -236,3 +240,31 @@ func TestUnitMapError(t *testing.T) {
 		t.Fatal("unexpected return value")
 	}
 }
+
+func TestUnitPadding(t *testing.T) {
```

Comment:
It could be beneficial if @xhdix could take a look at this test and double check